ui/conf/nginx.conf

```diff
@@ -0,0 +1,51 @@
+server {
+    listen       80;
+    server_name  localhost;
+
+    #charset koi8-r;
+    #access_log  /var/log/nginx/host.access.log  main;
+
+    location / {
+        root   /usr/share/nginx/html/sup;
+        index  index.html;
+    }
+
+    location ~ /brand/(.+) {
+        root /usr/share/nginx/html/sup;
+
+        try_files /override-brand/$1 /scality/$1;
+    }
+
+    #error_page  404              /404.html;
+
+    # redirect server error pages to the static page /50x.html
+    #
+    error_page   500 502 503 504  /50x.html;
+    location = /50x.html {
+        root   /usr/share/nginx/html;
+    }
+
+    # proxy the PHP scripts to Apache listening on 127.0.0.1:80
```

Comment:
No need to keep these comments around, I'd say.

Reply:
I just removed those comments.